packages/ui/src/components/ContinueWithUser.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import { colors, type, layout } from "../styles";
+import Button from "./Button";
+import { IconName } from "./IconShared";
+import Logo from "./Logo";
+import Spacer from "./Spacer";
+
+export interface ContinueWithUserProps {
+  colorPalette: colors.ColorPalette;
+  email: string | null;
```

Comment:
Would you mind making this field required and throwing in the parent if `email` ends up `null`? If that happens I'd like to make sure it sounds alarms in Sentry. They won't be able to proceed anyway!